src/components/NewWorkspaceModal.ts

```diff
@@ -80,6 +83,9 @@ export interface NewWorkspaceModalProps {
   requiredAuthDomain?: string;
   requireEnhancedBucketLogging?: boolean;
   title?: string;
+  waitForServices?: {
```

Comment:
My thinking here is that we'd eventually want to add support for CBAS as well. It looks like it has a similar issue and importing into a new Azure workspace is disabled when importing a workflow.